CONTRIBUTING.md

```diff
@@ -0,0 +1,160 @@
+
+### Never made an open source contribution before? Wondering how contributions work in our project? Here's a quick rundown!
+
+### Take a look at the Existing Issues or create your own Issues!
+- Wait for the Issue to be assigned to you after which you can start working on it.
+- Fork the Repo and create a Branch for any Issue that you are working upon.
+- Read the Code of Conduct
+- Create a Pull Request which will be promptly reviewed and suggestions would be added to improve it.
+
+## How to make a Pull Request
+If you think that you can add a new feature or want to fix a bug. You can start contributing by following the below instructions:
+
+1. Create a folder at your desired location (usually at your desktop).
+
+2. Open Git Bash Here
+
+3. Create a Git repository.
+
+**Run Command**
+
+```
+git init
+```
+
+## Fork this repository
+
+* Fork this repository by clicking on the fork button on the top of this page.
+This will create a copy of this repository in your account.
+
+## Clone the repository
+
+* Now clone the forked repository to your machine. Go to your GitHub account, open the forked repository, click on the code button and then click the _copy to clipboard_ icon.
+
+Open a terminal and run the following git command:
+
+```
+git clone "url you just copied"
+```
+
+where "url you just copied" (without the quotation marks) is the url to this repository (your fork of this project). 
+
+For example:
+
+```
+git clone https://github.com/user-name/PDF-Dark-Mode.git
+```
+
+where `user-name` is your GitHub username. Here you're copying the contents of the PDF-Dark-Mode repository on GitHub to your computer.
+
+## Navigate to the project directory
+* After cloning the project in your computer, navigate to the project file using the command below.
+```
+cd PDF-Dark-Mode
+```
+
+## Add a reference(remote) to the original repository
+
+```
+git remote add upstream https://github.com/PDF-Dark-Mode.git
+```
+
+* Check the remotes for this repository.
+
+```
+git remote -v
+```
+
+* Always take a pull from the upstream repository to your master branch to keep it at par with the main project(updated repository).
+
+```
+git pull upstream main
+```
+
+## Create a new branch
+
+```
+git checkout -b <your_branch_name>
+```
+
+* Perform your desired changes to the code base.
+
+1. Check you changes.
+
+```
+git status
+```
+
+```
+git diff
+```
+
+2. Stage your changes
+
+```
+git add . <\files_that_you_made_changes>
+```
+
+3. Commit your changes.
+
+```
+git commit -m "Relevant message"
+```
+
+4. Push the committed changes in your feature branch to your remote repo.
+
+```
+git push -u origin <your_branch_name>
+```
+
+5. To create a pull request, click on `compare and pull request.
+
+6. Add an appropriate title and description to your PR explaining your changes.
+
+7. Cick on Create pull request. 
+
+## Setup the project in your Local Mahine:
```

Comment:
typo "Mahine" .... please set up the project on your machine and run it on your browser successfully and write the methods you followed to achieve that .... I have added that on README as well .. you can use that as reference